client/src/App.css

```diff
@@ -1,3 +1,7 @@
+* {
```

Comment:
Because we're using makeStyles in each component, can we do away with this file completely?

Reply:
True.  I've been looking it up but haven't been able to find a way to recreate something like this in Material-UI:
* { box-sizing: border-box }

Reply:
Can we add it to the theme? If not maybe using this would be more consistent with Material UI convention - https://material-ui.com/components/css-baseline/